flask_monitoringdashboard/test/views/test_status_code_tracking.py

```diff
@@ -0,0 +1,137 @@
+import unittest
+from flask import Flask, json, jsonify
+from flask_monitoringdashboard.core.cache import EndpointInfo
+from flask_monitoringdashboard.database import session_scope, Request
+
+from flask_monitoringdashboard.test.utils import (
+    set_test_environment,
+    clear_db
+)
+
+
+def get_test_app_for_status_code_testing(schedule=False):
+    """
+    :return: Flask Test Application with the right settings
+    """
+    import flask_monitoringdashboard
+
+    app = Flask(__name__)
+
+    @app.route('/return-a-simple-string')
+    def return_a_simple_string():
+        return 'Hello, world'
+
+    @app.route('/return-a-tuple')
+    def return_a_tuple():
+        return 'Hello, world', 404
```

Comment:
don't use 404 here. You can add a test for checking the 404 for an unknown endpoint.

Reply:
But requests to non-existing endpoints aren't logged. What would you test? That there are no requests logged? That would be outside of the scope of this test.

Reply:
This test covers the case where a user manually return a status code. 

And this happens. Imagine that someone requests `/posts/125` and there is no post with ID=125. In this case it would be appropriate to return status code 400.

Reply:
ah, oke. I forgot that we don't monitor 404 endpoints ;) Now it makes sense